binary_to_decimal/binary_to_decimal.js

```diff
@@ -0,0 +1,53 @@
+// ----------------------------------
+// Function binaryToDecimal
+
```

Comment:
Celý soubor nemá žádná pravidla pro formátování a je to celé nepěkné, obzvlášť, když by s tím měl pracovat další programátor. Mrkni co jsou linters. Jedná se o utility, která ti formátuje kód a drží jednotný systém po celém tvém projektu. Jestli používáš VS Code, tak vyzkoušej plugin Prettier, jestli máš jiný IDE, mrkni na net :)

Není to věc funkčnosti, ale estetiky. Až budeš udržovat tisíce řádků kódu v projektu, sám to oceníš.

Reply:
Super! Toto je vec, ktorú budem určite používať...